### PR TITLE
move to the upstream helm

### LINKED ISSUE #1443: Move from a proprietary helm fork to the upstream helm repository
Originally, some of the helm repository methods were under experimental/internal distribution. Therefore, Fybrik used a proprietary helm fork. Recently, helm was reorganized, and it allows using the upstream repository directly.

Elimination of the private helm fork, will significantly simplify the Fybrik release process, help easily include helm updates, and eventually increase Fybrik robustness.

### PR DESCRIPTION
**Pull request by roytman**
Move to the upstream helm repo.

Bump to upstream helm 3.8.2 plus update a list of other third-party dependencies.

/closes #1443
/closes #1219

Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

### CODE REVIEW

File: hack/tools/install_helm.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 source ./common.sh
 
-DESIRED_VERSION=v3.7.2
+DESIRED_VERSION=v3.8.1
```
Comment (shlomitk1) [inline hack/tools/install_helm.sh:8]: It's 3.8.2 in go.mod - does it matter?

Comment (roytman) [reply hack/tools/install_helm.sh:8]: Thanks, fix it

**Review by shlomitk1 — DISMISSED** [review]

File: hack/make-rules/helm.mk [hunk 2]
```diff
@@ -22,7 +22,6 @@ HELM_TAG ?= 0.0.0
 
 CHART_REGISTRY_PATH := oci://${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}
 
-export HELM_EXPERIMENTAL_OCI=1
 export GODEBUG=x509ignoreCN=0
```
Comment (revit13) [inline hack/make-rules/helm.mk:25]: for versions lower than 3.8 HELM_EXPERIMENTAL_OCI is needed and as `DESIRED_VERSION` can  changed I suggest keeping HELM_EXPERIMENTAL_OCI

Comment (roytman) [reply hack/make-rules/helm.mk:25]: Why will somebody use the old version?

**Review by revit13 — APPROVED** [review]

**Pull request merged**